doc/vim-instant-markdown.txt

```diff
@@ -22,6 +22,7 @@ CONTENTS                                           *vim-instant-markdown-content
    3.9. g:instant_markdown_browser...................|g:instant_markdown_browser|
    3.10. g:instant_markdown_port........................|g:instant_markdown_port|
    3.11. g:instant_markdown_autoscroll............|g:instant_markdown_autoscroll|
+   3.12. g:instant_markdown_theme......................|g:instant_markdown_theme|
```

Comment:
Can you add a description in the documentation, in the same document, after this section?

https://github.com/instant-markdown/vim-instant-markdown/blob/383aa2900f855eddc42ffbe9d636c54e2f2939d5/doc/vim-instant-markdown.txt#L229-L239